handlers/sf-move-subscriptions-api/src/main/scala/com/gu/sf/move/subscriptions/api/SFMoveSubscriptionsApiRoutes.scala

```diff
@@ -11,32 +11,43 @@ import org.http4s.{DecodeFailure, HttpRoutes, Request, Response}
 
 object SFMoveSubscriptionsApiRoutes extends LazyLogging {
 
-  def apply[F[_]: Effect](): HttpRoutes[F] = {
+  def apply[F[_] : Effect](moveSubscriptionService: SFMoveSubscriptionsService[F]): HttpRoutes[F] = {
     object http4sDsl extends Http4sDsl[F]
     import http4sDsl._
 
-    val rootInfo = MoveSubscriptionApiRoot("This is the sf-move-subscriptions-api")
+    val selfDoc = MoveSubscriptionApiRoot(
+      description = "This is the sf-move-subscriptions-api",
+      exampleRequests = List(
+        ExampleReqDoc(
+          method = "POST",
+          path = "subscription/move",
+          body = MoveSubscriptionReqBody(
+            zuoraSubscriptionId = "Zuora Subscription Id",
+            sfAccountId = "SF Account Id",
+            sfFullContactId = "SF Full contact Id",
+          )
+        )
+      )
+    )
 
     def handleMoveRequest(request: Request[F]): F[Response[F]] = {
       (for {
         reqBody <- request.attemptAs[MoveSubscriptionReqBody]
           .leftMap { decodingFailure: DecodeFailure =>
             BadRequest(MoveSubscriptionApiError(s"Failed to decoded request body: $decodingFailure"))
           }
-        resp <- SFMoveSubscriptionsService.moveSubscription(reqBody)
+        resp <- moveSubscriptionService.moveSubscription(reqBody)
           .leftMap(
             error =>
-              InternalServerError(MoveSubscriptionApiError(s"Failed create voucher: $error"))
-          )
+              InternalServerError(MoveSubscriptionApiError(s"Failed to move subscription: $error"))
+          ).map(serviceRes => MoveSubscriptionApiSuccess(serviceRes.message))
```

Comment:
Ive been putting lots of thought into making these routes as clear as possible, maybe:
```suggestion
         .bimap(
            error => InternalServerError(MoveSubscriptionApiError(s"Failed to move subscription: $error")),
            serviceRes => Ok(MoveSubscriptionApiSuccess(serviceRes.message))
          )
      } yield resp).merge.flatten
```

Reply:
That looks much better !
Will do that
Thanks for the review :+